fetchTMDBID.js

```diff
@@ -0,0 +1,59 @@
+import fetch from "node-fetch";
+import { PrismaClient } from "@prisma/client";
+import Bottleneck from "bottleneck";
+import dotenv from "dotenv";
+dotenv.config();
+
+const prisma = new PrismaClient();
+const apiKey = process.env.VITE_API_KEY; // Ensure your API key is loaded from .env
+
+// Initialize the limiter to handle up to 50 requests per second
+const limiter = new Bottleneck({
+  minTime: 100,
+  maxConcurrent: 1,
+  highWater: 10000,
+});
+
+async function fetchAndUpdateMovie(movie) {
+  const cleanedTitle = movie.title.slice(0, -6);
```

Comment:
just curious why the title needs to be cleaned?

Reply:
So the titles in the database include a data year (e.g. (2004)) which messes up the search query when trying to find TMDB IDs